contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -209,39 +212,63 @@ func (t *T) Reset() {
 	t.Spec = Spec{}
 }
 
+func (t *T) computeResourceLimit(name string, defaultLimit float64) float64 {
+	var x float64 = 0.0
+	for _, container := range t.Pod.Spec.Containers {
+		for limitName, limit := range container.Resources.Limits {
+			if string(limitName) == name {
+				value := limit.Value()
+				if value > 1000000 { // avoid overflow with MilliValue
+					x += float64(value)
+				} else {
+					x += float64(limit.MilliValue()) / 1000.0
+				}
+			}
+		}
+	}
+	if x == 0.0 {
```

Comment:
actually if a pod has a container that doesn't specify limits, then the limits for the entire pod are unbounded.

see (and perhaps reuse?) https://github.com/GoogleCloudPlatform/kubernetes/blob/a8269e38c9e2bf81ba18cd6420e2309745d5b0b9/pkg/resourcequota/resource_quota_controller.go#L248